src/main/java/net/minecraftforge/common/extensions/IForgeEntityMinecart.java

```diff
@@ -37,22 +39,26 @@
 
 public interface IForgeEntityMinecart
 {
-    public static float DEFAULT_MAX_SPEED_AIR_LATERAL = 0.4f;
-    public static float DEFAULT_MAX_SPEED_AIR_VERTICAL = -1.0f;
-    public static double DEFAULT_AIR_DRAG = 0.95f;
-    public static IMinecartCollisionHandler COLLISIONS = null;
+    float DEFAULT_MAX_SPEED_AIR_LATERAL = 0.4f;
+    float DEFAULT_MAX_SPEED_AIR_VERTICAL = -1.0f;
+    double DEFAULT_AIR_DRAG = 0.95f;
+    double DEFAULT_GROUND_DRAG = 0.5d;
 
     default EntityMinecart getMinecart() {
         return (EntityMinecart)this;
     }
 
+    static void setCollisionHandler(IMinecartCollisionHandler handler) {
+        ExtensionFields.minecartCollisionHandler = handler;
+    }
+
     /**
      * Gets the current global Minecart Collision handler if none
      * is registered, returns null
      * @return The collision handler or null
      */
-    default IMinecartCollisionHandler getCollisionHandler() {
-        return COLLISIONS;
+    default  @Nullable IMinecartCollisionHandler getCollisionHandler() {
```

Comment:
Not `static` as to save patch size, or I need to call
`net.minecraftforge.common.extensions.IForgeEntityMinecart.getCollisionHandler()` instead.